awscli/customizations/wizard/wizards/lambda/new-function.yml

```diff
@@ -42,16 +48,55 @@ plan:
         type: prompt
         description: Select the role to use
         choices: existing_roles
+  handler:
+    shortname: Handler
+    description: Provide handler and code location
+    values:
       handler:
         type: prompt
         description: Enter the handler for your function
       zip_filename:
-        type: fileprompt
-        description: Enter the new location of your code zip file
+        type: prompt
+        completer: file_completer
+        description: Enter location of your code zip file
+  preview:
+    shortname: Preview
+    description: Preview results
+    values:
+      preview_cli_command_value:
+        type: template
+        value: |
+          aws lambda create-function \
+            --function-name '{function_name}' \
+            --role '{role_arn}' \
+            --runtime '{runtime}' \
+            --handler '{handler}' \
+            --zip-file fileb://{zip_filename}
+      preview_value:
+        type: template
+        value: |
+          {%if {preview_type} == preview_cli_command %}
+          {preview_cli_command_value}
+          {% endif %}
+      preview_type:
+        type: prompt
+        description: Select an preview format
+        choices:
+          - display: None
+            actual_value: preview_none
+          - display: AWS CLI command
+            actual_value: preview_cli_command
```

Comment:
I'm wondering, if we should also generate CloudFormation snippets for this. My guess is that we did not do it because the zip file with the code would need to be uploaded to s3. But I'm wondering if we just generate one anyway and just add placeholders for the source code part informing the user that they need to update the values to whatever S3 bucket they want to use? The nice part is that because we are using yaml, we can embed comments to elaborate more on the placeholders if we want.